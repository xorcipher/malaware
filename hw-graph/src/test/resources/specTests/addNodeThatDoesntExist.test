# Tests that a node can be added to a graph when the node does not already exist in the graph.

# Adds a new node to a graph.

CreateGraph graph

AddNode graph A