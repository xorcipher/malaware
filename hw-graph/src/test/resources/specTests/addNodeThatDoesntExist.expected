# Tests that a node can be added to a graph when the node does not already exist in the graph.

# Adds a new node to a graph.

created graph graph

added node A to graph